src/igvfd/schemas/technical_sample.json

```diff
@@ -0,0 +1,96 @@
+{
+    "title": "Technical Sample",
+    "id": "/profiles/technical_sample.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a specific technical sample.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab"
```

Comment:
@KAndreeva Should we require type as well?

Reply:
In technical sample, the sample material is now a required field.